purchase_order_weight_volume/models/purchase_order.py

```diff
@@ -26,15 +26,22 @@ class PurchaseOrder(models.Model):
     total_weight = fields.Float(
         compute="_compute_total_physical_properties",
         digits="Stock Weight",
+        store=True,
```

Comment:
Already added in the dependency PR: https://github.com/OCA/purchase-workflow/blob/ce3c13c7057fce8af43d79ecd34ab97b9172ca2f/purchase_order_weight_volume/models/purchase_order.py#L29